apps/explorer/src/pages/checkpoints/CheckpointsTable.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { useRpcClient } from '@mysten/core';
+import { ArrowRight12 } from '@mysten/icons';
+import { useQuery } from '@tanstack/react-query';
+import { useMemo, useState } from 'react';
+
+import Pagination from '~/components/pagination/Pagination';
+import TabFooter from '~/components/tabs/TabFooter';
+import {
+    statusToLoadState,
+    type PaginationType,
+} from '~/components/transaction-card/RecentTxCard';
+import { TxTableCol } from '~/components/transaction-card/TxCardUtils';
+import { TxTimeType } from '~/components/tx-time/TxTimeType';
+import { CheckpointLink } from '~/ui/InternalLink';
+import { Link } from '~/ui/Link';
+import { PlaceholderTable } from '~/ui/PlaceholderTable';
+import { TableCard } from '~/ui/TableCard';
+import { Text } from '~/ui/Text';
+
+interface CheckpointsTableProps {
+    itemsPerPage?: number;
+    currentPage?: number;
+    paginationType?: PaginationType;
+    handlePageChange?: (page: number) => void;
+    shouldRefetch?: boolean;
+    refetchInterval: number;
+}
+
+export function CheckpointsTable({
+    itemsPerPage: initialItemsPerPage,
```

Comment:
nit: maybe call this prop `initialItemsPerPage` so it isn't misleading / you don't need to re-name it here?

Reply:
more or less :) it was previously just a table of mock data - until we can see the start / end sequence numbers in an epoch (coming soon) we can't implement the checkpoints table